configs/configStore-sample.json

```diff
@@ -3,8 +3,8 @@
   "Storetype": "config",
   "Store": {
     "Device1Version": {
-      "Name": "Device1Version",
-      "Device": "Device1",
```

Comment:
I didn't realize you had rename this - can we keep the Device1 and add on localhost as an extra config

I think it's valuable to have Device1 and Device2 configs as well as other